src/models/users.js

```diff
@@ -70,13 +70,26 @@ const initialize = (models) => {
     },
   );
   models.users.hasMany(
-    models.wanted_contents, {
+    models.wantedContents, {
       as: 'user_wanted_contents',
       foreignKey: 'user_id',
       sourceKey: 'id',
     },
   );
-
+  models.users.hasOne(
```

Comment:
buraya göre bir user sadece 1 mesaj atabilir, bu sistemimiziçin doğdu değil bir user'aait birden fazla mesaj bulunabilir